sdk/src/beta9/runner/container.py

```diff
@@ -50,12 +51,25 @@ async def _run():
                     print(line.strip().decode("utf-8"))
 
                 if not self.killed:
-                    stub.end_task(
-                        EndTaskRequest(
+                    end_task_and_send_callback(
```

Comment:
shouldn't we just send the callback regardless?

Reply:
yea so if it was `killed` meaning it was canceled, we send a different callback with task status cancelled